import-service/google-project.tf

```diff
@@ -11,6 +11,48 @@ resource "google_project_iam_custom_role" "cloud-scheduler-appengine-custom-role
                   "cloudscheduler.jobs.update", "cloudscheduler.locations.get", "cloudscheduler.locations.list"]
 }
 
+# key-rotation policy for service accounts
+  # note this requires the terraform to be run regularly
+  resource "time_rotating" "sa_key_rotation_policy" {
+    rotation_days = 75 // compliance requirement: 90 days
+  }
+
+  // create deployer SA
+  resource "google_service_account" "sa_deployer" {
```

Comment:
Mini-nit-pick -- but would it be worthwhile to also include a mini `description` argument in this resource too?